_posts/faq.md

```diff
@@ -60,6 +60,7 @@ Eventuali annunci di posizioni aperte, presentazioni di conferenze, o altri even
 - vi consigliamo di rimanere a disposizione per eventuali domande di persona, al termine del incontro
 - il numero massimo di sessioni/annunci durante l' open-mic per ciascun evento, è gestito a discrezione dei coordinatori del gruppo
 
+Per alcune tipologie di incontri (online/in presenza/...) o in alcune location, potrebbero essere previste ulteriori guidelines da dover rispettare.
```

Comment:
location -> sedi
guidelines -> linee guida

Reply:
sostituisco

giusto per la "cronaca":
https://github.com/jugmilano/jugmilano.github.io/blob/f7191d4fa67abd0a4315af5749b059473f867ae8/_posts/faq.md?plain=1#L11

sara' il Lunedi' ma non mi viene in mente un sinonimo in Italiano di `online`

Reply:
Penso si possa tradurre come "su internet", ma probabilmente la cosa più *furba* è restare vaghi e rimuovere questi dettagli, che comunque aggiungono poco:

Per alcuni incontri o in alcune sedi potrebbero essere previste ulteriori linee guida da rispettare.